jupyter-on-gke/persistent_storage_deployments/filestore-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: filestore-claim
+  namespace: <NAMESPACE>
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: ""
```

Comment:
storageClassName: filestore-storage-class

(which is created in the filestore-storage-class.yaml)